handlers/index.js

```diff
@@ -0,0 +1,199 @@
+const Session = require("../session/session");
+const SessionStorage = require("../session/session_storage");
+const { v4: uuidv4 } = require("uuid");
+const logger = require("../logger");
+const {
+    FdkSessionNotFoundError,
+    FdkInvalidOAuthError,
+} = require("../error_code");
+
+const fpInstall = async (company_id, application_id, ext) => {
```

Comment:
Add named function here as it will be helpful while reading stacktrace.

Reply:
Resolved